IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ElectricFlowStorageDeviceTypeUPS/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of uninterruptible power supply equipment used in railway. The property set can be used by the predefined type UPS  of IfcElectricFlowStorageDevice.
```

Comment:
Only railway?

Reply:
Not only. 0871f7b28802c8b6bf8a2b767b0356cb11e25edd